File: ANSv4/Catalogo4/Global01.cs
using System;

namespace Catalogo
{

    static class Global01
    {

       public enum TiposDeCatalogo
        {
            Invitado = 1,
            Cliente = 2,
            Viajante = 3,
            Supervisor = 4
        }

       public static string VersionApp;
       public static TiposDeCatalogo miSABOR;

       public const string up2014Ad = ";User Id=hpcd-rw;Password=data700mb";
       public const string up2014Us = ";User ID=inVent;Password=video80min";

       public static System.Data.OleDb.OleDbConnection Conexion;
       public static System.Data.OleDb.OleDbTransaction TranActiva;
       public static string AppPath;
       public static string PathAcrobat;
       public static string FileBak;
         
       public static string cstring;        
       public static string dstring;
       public static string sstring;

       public static string strConexionUs;
       public static string strConexionAd;

       public static string IDMaquina;
       public static string IDMaquinaCRC;
       public static string IDMaquinaREG;
       public static string LLaveViajante;
       public static bool RecienRegistrado;
       public static bool AppActiva;

       public static string NroUsuario;
       public static System.DateTime dbCaduca;
       public static System.DateTime appCaduca;
       public static System.DateTime F_ActCatalogo;
       public static System.DateTime F_ActClientes;
       public static System.DateTime F_UltimoAcceso;
       public static bool ActualizarClientes;

       public static bool EnviarAuditoria;
       public static bool AuditarProceso;
       public static bool xError;
       public static string URL_ANS;
       public static string URL_ANS2;
       public static string MainWindowCaption;
       public static bool NoConn;
       public static int MiBuild;
       public static byte ListaPrecio;
       public static float Dolar;
       public static float Euro;

       public static string Cuit;
       public static string pin;
       public static string RazonSocial;
       public static string ApellidoNombre;

       public static string NroImprimir;

    }
}

File: ANSv4/CatalogoLibraryVB/modules/Global01.vb
Module Global01


    '---------------------------------------------------------------
    ' Define como se llama este modulo para el control de errores
    Private Const m_sMODULENAME_ As String = "MainMod"

    Public Const u2String As String = "?"
    Public Const c2String As String = "Provider=Microsoft.Jet.OLEDB.4.0;Data Source=C:\Catalogo ANS\Datos\catalogo.mdb;Persist Security Info=True;Password=video80min;User ID=inVent;Jet OLEDB:System database=C:\Windows\Help\kbappcat.hlp"

    ' VARIABLES GLOBALES
    Public Structure vAPP


        Public IDMaquina As String 'vg.misabor & obtenerCRC(WMI)
        Public IDMaquinaCRC As String 'obtenerCRC(vg.IDMaquina)
        Public IDMaquinaREG As String 'obtenerCRC(vg.IDmaquina & vg.IDmaquinaCRC)
        Public LLaveViajante As String
        Public AppActiva As Boolean

        Public Conexion As System.Data.OleDb.OleDbConnection

        Public Cuit As String
        Public RazonSocial As String
        Public ApellidoNombre As String
        Public Domicilio As String
        Public Telefono As String
        Public Ciudad As String
        Public Email As String
        Public NroUsuario As String
        Public NroPDR As String
        Public NroImprimir As String
        Public dbCaduca As Date
        Public appCaduca As Date
        Public FechaUltimoAcceso As Date
        Public F_ActCatalogo As Date
        Public F_ActClientes As Date
        Public EnviarAuditoria As Boolean
        Public AuditarProceso As Boolean
        Public IpSettingIni As Boolean
        Public EsGerente As Boolean
        Public LoginSucceeded As Boolean
        Public ActualizarCatalogo As Boolean
        Public ActualizarCuentas As Boolean
        Public RecienRegistrado As Boolean
        Public xError As Boolean ' Se tuvo que cambiar el nombre de error a err        
        Public TranActiva As System.Data.OleDb.OleDbTransaction
        Public auditor As Auditor
        Public URL_ANS As String
        Public URL_ANS2 As String
        Public ExploradorCaption As String
        Public ValeCombo As Boolean
        Public NoConn As Boolean
        Public Path As String
        Public MiBuild As Integer
        Public ListaPrecio As Byte
        Public FileBak As String
        Public PathAcrobat As String
        Public Dolar As Single
        Public Euro As Single
    End Structure

    Public cstring As String = ""
    Public qstring As String = ""
    Public dstring As String = ""
    Public sstring As String = ""
    Public ArchCerradura As String = ""
    Public ArchLlave As String = ""

    Public vg As vAPP

End Module
